README.md

```diff
@@ -36,16 +30,4 @@ Project resources
 - [Gitter.im Chat Room](https://gitter.im/Netflix/security_monkey)
 - [CloudAux](https://github.com/Netflix-Skunkworks/cloudaux)
 - [PolicyUniverse](https://github.com/Netflix-Skunkworks/policyuniverse)
-- [Troubleshooting](docs/troubleshooting.md)
-
-
-Instance Diagram
```

Comment:
I would prefer to keep this here on the main page.

Reply:
Done: https://github.com/Netflix/security_monkey/pull/950/commits/51c9079f5b3080a0f78bf978725074d0dd47e38c